source/concepts/applications.rst

```diff
@@ -0,0 +1,746 @@
+to particles on-the-fly.
+
+The iMD application defines how to send user interactions to the server, the
+expected behaviour of the server regarding these interactions, and how the
+server can communicate the result of these interactions on the simulation to
+the clients.
+
+The application assumes it is used in conjunction with the :ref:`trajectory
+application <trajectory-application>` or a similar enough application to share
+the simulation itself.
+
+A user sends an interaction as a point of origin, the particles to which it
+applies and a set of parameters. The server, then collects all the user
+interactions, computes the corresponding forces and propagates them with the
+other forces in the simulation.
+
+The interactions can use different :ref:`equations <force-equations>` to
+compute the force :math:`\mathbf{F}_{\text{COM}}` to the center of mass of the group of
+target particles. The force is then distributed to each particles differently
+is the interaction is mass weighted of not. If if it mass weighted, then the
+force :math:`\mathbf{F}_i` applied to the particle :math:`i` is :math:`\mathbf{F}_i = s \cdot m_i
+\frac{\mathbf{F}_{\text{COM}}}{N}` with :math:`s` a scaling factor set by the user,
+:math:`m_i` the mass of particle :math:`i`, and :math:`N` the number of target
+particles for the interaction. If the interaction is not mass weighted, then
+:math:`\mathbf{F}_i = s \cdot \frac{\mathbf{F}_{\text{COM}}}{N}`. Finally, :math:`|\mathbf{F}_i|` can be
+capped to a maximum value specified by the user to avoid applying too large
+forces.
+
+Each interaction type also define the equation for the energy
+:math:`E_{\text{COM}}`. For mass weighted interaction, the energy for the
+interaction is :math:`E = \frac{E_{\text{COM}}}{N}\sum_{i=0}^{N}m_i`. For non
+mass weighted :math:`E = E_{\text{COM}}`.
+
+.. _force-equations:
+
+Force equations
+~~~~~~~~~~~~~~~
+
+Each server is free to implement the interaction equation they choose. However,
+there are some that are commonly implemented: the Gaussian force, the harmonic
+force, and the constant force. They all depend on the vector :math:`\mathbf{d}` between
+the origin of the interaction, :math:`\mathbf{r}_{\text{user}}`, and the center of mass
+of the set of target particles :math:`\mathbf{r}_{\text{COM}}`. So, :math:`\mathbf{d} =
+\mathbf{r}_{\text{user}} - \mathbf{r}_{\text{COM}}`.
+
+The Gaussian force is defined by:
+
+.. math::
+
+   \begin{align}
+      \mathbf{F}_{\text{COM}}^{\text{Gaussian}} &= -\frac{\mathbf{d}}{\sigma^2}\exp{-\frac{| \mathbf{d} | ^2}{2\sigma^2}} \\
+      E_{\text{COM}}^{\text{Gaussian}} &= \exp{-\frac{| \mathbf{d} |^2}{2\sigma^2}}
+   \end{align}
+
+with :math:`\sigma = 1`. With this force, the user interaction is stronger when
+applied close to the particles.
+
+The harmonic force is defined by:
+
+.. math::
+
+   \begin{align}
+   \mathbf{F}_{\text{COM}}^{\text{Harmonic}} &= -k \mathbf{d} \\
+   E_{\text{COM}}^{\text{Harmonic}} &= \frac{1}{2}k| \mathbf{d} |^2
```

Comment:
I believe that the energy of the user should be the _negative_ of the exponential defining the Gaussian force
```suggestion
   E_{\text{COM}}^{\text{Harmonic}} &= - \frac{1}{2}k| \mathbf{d} |^2
```
as per the source code: https://github.com/IRL2/nanover-protocol/blob/main/python-libraries/nanover-core/src/nanover/imd/imd_force.py#L223